src/megatron/bridge/recipes/qwen_vl/qwen25_vl_step.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
should we offer higher-level functions like `pretrain_gpt` / `finetune_gpt` / `pretrain_qwen25_vl`, `finetune_qwen25_vl` that automatically select the right step for users? 
or how else should we associate the custom forward step func with the recipe? otherwise it might not be obvious to users that an implementation exists

previously this was bound to the model config, but we want to keep the model providers independent of the training side